src/views/Batches/batchComponents/BatchForm.jsx

```diff
@@ -121,6 +122,12 @@ const BatchForm = (props) => {
         // otherwise return only those status that are applicable to all
         return status['applicable-to'] === 'All';
       });
+
+      setPaymentOptions(data.mode_of_payment.map(mode_of_payment => ({
```

Comment:
I think it will be good if we stick to one type of naming convention. Let's go with camelCasing
```suggestion
      setPaymentOptions(data.mode_of_payment.map(modeOfPayment => ({
```